### PR TITLE
Feature/sequelize

### PR DESCRIPTION
**Pull request by menaka809**
Use sequelize ORM library when creating a database connection for PostgreSQL database.

### CODE REVIEW

File: pet-server/server-logger-inf.log
Comment (nimi-kaushal) [inline pet-server/server-logger-inf.log:1]: add to git ingore

Comment (menaka809) [reply pet-server/server-logger-inf.log:1]: added server-logger-inf file to the git ingore

File: pet-server/models/companyModel.js [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    country: {
+      type: DataTypes.STRING,
+      validate: {
+        len: [2, 50],
+      },
+    },
+  },
+  {
+    tableName: "company",
+  }
```
Comment (nimi-kaushal) [inline pet-server/models/companyModel.js:75]:
```
      sequelize,
      tableName: 'company',
      underscored: true
```

File: pet-server/models/companyModel.js
Comment (nimi-kaushal) [inline pet-server/models/companyModel.js:1]: use camel case for attributes

File: pet-server/models/companyPaySlipInfoModel.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+      },
+    },
+    payday: {
+      type: DataTypes.DATE,
+      allowNull: false,
+    },
+  },
+  {
+    tableName: "companyPaySlipInfo",
+  }
```
Comment (nimi-kaushal) [inline pet-server/models/companyPaySlipInfoModel.js:37]:
```
      sequelize,
      tableName: 'company_pay_slip_info',
      underscored: true
```

File: pet-server/models/usersModel.js [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+      type: DataTypes.STRING,
+      allowNull: false,
+      validate: {
+        len: [8, 72],
+      },
+    },
+  },
+  {
+    tableName: "users",
+  }
```
Comment (nimi-kaushal) [inline pet-server/models/usersModel.js:47]: sequelize,
      tableName: 'user',
      underscored: true

**Review by nimi-kaushal — CHANGES_REQUESTED** [review]

File: pet-server/models/companyPaySlipInfoModel.js (hunk 2 above)
Comment (menaka809) [reply pet-server/models/companyPaySlipInfoModel.js:37]: enable underscored true , added single quote, and use camel case. push the changes

File: pet-server/models/usersModel.js (hunk 3 above)
Comment (menaka809) [reply pet-server/models/usersModel.js:47]: enable underscored true and added single quote. push the changes

File: pet-server/models/companyModel.js (hunk 1 above)
Comment (menaka809) [reply pet-server/models/companyModel.js:75]: change attribute name to the  camel case and push the changes.

File: pet-server/models/companyModel.js
Comment (menaka809) [reply pet-server/models/companyModel.js:1]: change the attribute name to the camel case and push the changes.

**Review by nimi-kaushal — APPROVED** [review]

**Pull request merged**